lib/cmds/files_cmds/upload.js

```diff
@@ -197,4 +210,44 @@ exports.handler = async argv => {
       fs.unlinkSync(file);
     }
   }
+
+  function getPathToStrip (stripPath, filePath, isDir) {
+    if (!stripPath) {
+      return '';
+    }
+
+    const baseFileName = normalizeFileName(filePath);
+    let lastSepPath = baseFileName.lastIndexOf('/');
+
+    if (isDir) {
+      lastSepPath = baseFileName.length;
+    }
+
+    const pathToStrip = filePath.substring(0, lastSepPath + 1);
+
+    console.log('filePath=' + filePath + '\npathToStrip = ' + pathToStrip);
```

Comment:
Remove, or we also make use of the `debug` module and you could update these to use that.

Reply:
see above - using debug